test/helpers/handler.js

```diff
@@ -24,6 +24,7 @@ const callHandler = async function (t, handlers, name, event) {
 // Temporarily mock `console.log()` in order to retrieve the logs
 const mockConsoleLog = function () {
   const oldConsoleLog = console.log
+  // eslint-disable-next-line import/no-named-as-default-member
```

Comment:
Using a named import does not work with `sinon`, but `eslint-plugin-import` still seems to complain. I am not sure whether this a bug in `sinon` or in `eslint-plugin-import`.